youtube_transcript_api/test/test_formatters.py

```diff
@@ -28,6 +29,25 @@ def test_base_formatter_format_call(self):
         with self.assertRaises(NotImplementedError):
             Formatter().format_transcripts([self.transcript])
 
+    def test_srt_formatter(self):
+        content = SRTFormatter().format_transcript(self.transcript)
+        lines = content.split('\n')
+
+        # test starting lines
+        self.assertEqual(lines[0], "1")
+        self.assertEqual(lines[1], "00:00:00,000 --> 00:00:01,500")
+
+        # test end lines
+        self.assertEqual(lines[-2], self.transcript[-1]['text'])
+        self.assertEqual(lines[-1], "")
```

Comment:
This probably should be two separate tests. It would also be great if you could test whether `{'text': 'line between', 'start': 1.5, 'duration': 2.0},` has been formatted correctly, since in this case `start + duration` is larger than the `start` of the next line.

Reply:
How would you recommend making that test? Like this?
```javascript
content = SRTFormatter().format_transcript(self.transcript)
lines = content.split('\n')
self.assertEqual(lines[6], "00:00:01,500 --> 00:00:02,500")
```